docs/guides/database-hosts.mdx

```diff
@@ -0,0 +1,59 @@
+import Admonition from '@theme/Admonition';
+
+# Database Hosts
+
+Database hosts allow to create per-server databases on the given host.
+
+## Database Configuration
+
+### Create Database Host user
+
+You will need a database user with the correct permissions before continuing any further.
+
+Run the following commands to quickly create a user for your new database host:
+
+```sql
+mysql -u root -p
+
+# Remember to change 'yourPassword' below to be a unique password
+# Replace 127.0.0.1 with your panel ip if your panel and wings are on different machines
+CREATE USER 'pelicanuser'@'127.0.0.1' IDENTIFIED BY 'yourPassword';
+GRANT ALL PRIVILEGES ON *.* TO 'pelicanuser'@'127.0.0.1' WITH GRANT OPTION;
+exit
```

Comment:
You should add `FLUSH PRIVILEGES;` for good mesure

Reply:
That's no longer needed.